src/Commands/NotificationQueuer.php

```diff
@@ -92,6 +100,57 @@ public function listen($media_type_id) {
     $this->io()->progressFinish();
   }
 
+  /**
+   * Removes notifications that are older than the entities that reference them.
+   *
+   * @param \Lullabot\Mpx\DataService\Notification[] $notifications
+   *   An array of notifications.
+   * @param string $media_type_id
+   *   The media type being imported.
+   *
+   * @return array
```

Comment:
This returns `\Lullabot\Mpx\DataService\Notification[]`? If we document that then IDEs can do autocomplete on the array contents automatically.